scripts/build-dot-flow.js

```diff
@@ -0,0 +1,33 @@
+var path = require('path'), fs=require('fs'), shell = require('shelljs');
+const sep = require('path').sep;
+
+function fromDir(startPath,filter){
+
+    console.log('Starting from dir '+startPath+sep);
+
+    if (!fs.existsSync(startPath)){
+        return;
+    }
+
+    var files=fs.readdirSync(startPath);
+    for(var i=0;i<files.length;i++){
+        var filename=path.join(startPath,files[i]);
+        var stat = fs.lstatSync(filename);
+        var n    = filename.indexOf("__tests__");
+        if ( (stat.isDirectory()) && (n===-1)) {
```

Comment:
remove extra space and unnecessary parenthesis, and add space around equality operator